site/profile/manifests/cvmfs.pp

```diff
@@ -3,8 +3,14 @@
   String $initial_profile,
   Array[String] $repositories,
   Array[String] $lmod_default_modules,
-) {
+  Array[String] $alien_cache_repositories,
```

Comment:
`$alien_cache_repositories`  is missing a default value in either common.yaml or here.

Reply:
`$repositories` is also missing a default value it seems

Reply:
`$repositories` is defined `data/software_stack/` by either `compute_canada.yaml` or `eessi.yaml`.

Reply:
Ok, so compute_canada.yaml is loaded by default!